lib/berlim/company_accounts/employee.ex

```diff
@@ -15,12 +15,20 @@ defmodule Berlim.CompanyAccounts.Employee do
     timestamps()
   end
 
+  @doc false
+  def changeset(employee, company, attrs) do
+    employee
+    |> cast(attrs, [:name, :internal_id, :active, :sector_id])
```

Comment:
se isso começar a crescer demais você pode criar um changeset privado e usar nos dois, com o que for comum, ou criar uma variável lá em cima

```elixir
@expected_fields [:name, :internal_id, :active, :sector_id]
```

e usar aqui

```elixir
employee
|> cast(attrs, @expected_fields)
...
```

Reply:
show!